app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -133,62 +198,7 @@ class MainActivity : ComponentActivity() {
 
 
 
-    @Composable
-    fun RecycleSearch(name: String, products: List<Product> = ArrayList<Product>()) {
-        var recyclable by remember{ mutableStateOf("")}
-        var location by remember{ mutableStateOf("")}
-        val context = LocalContext.current
-       // val i = ImageView(this).apply {
-       //     setImageResource(R.drawable.ic_hamburger_menu.)
-      //  }
-        //var hamburger_Menu = ImageView(this).apply{
-        //    setImageResource(R.drawable.ic_hamburger_menu)
-        //}
-        Row {
-            Arrangement.Center
-            ProfileMenu()
-        }
-
-        Column {
-
-            ProductSpinner(products = products)
-            //TextFieldWithDropdownUsage(dataIn = pr)
 
-            OutlinedTextField(
-                value = recyclable,
-                onValueChange = {recyclable = it},
-                label = { Text(stringResource(R.string.recyclable)) },
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
-                    .border(width = 1.dp,
-                            color = Color.Black,
-                            shape = RoundedCornerShape(4.dp)),
```

Comment:
The OutlinedTextField already has a border automatically. I the border modifier because it made your text field look funny with two borders